app/default_overrides.go

```diff
@@ -264,19 +264,19 @@ func DefaultEvidenceParams() *tmproto.EvidenceParams {
 
 func DefaultConsensusConfig() *tmcfg.Config {
 	cfg := tmcfg.DefaultConfig()
-	mempoolSize := int64(appconsts.DefaultUpperBoundMaxBytes) * 3
 	// Set broadcast timeout to be 50 seconds in order to avoid timeouts for long block times
 	cfg.RPC.TimeoutBroadcastTxCommit = 50 * time.Second
 	// this value should be the same as the largest possible response. In this case, that's
-	// likely Unconfirmed txs for a full mempool
-	cfg.RPC.MaxBodyBytes = mempoolSize
+	// likely Unconfirmed txs for a full mempool and a few extra bytes.
+	cfg.RPC.MaxBodyBytes = appconsts.MempoolSize + (mebibyte * 32)
 	cfg.RPC.GRPCListenAddress = "tcp://127.0.0.1:9098"
 
 	cfg.Mempool.TTLNumBlocks = 12
-	cfg.Mempool.TTLDuration = 75 * time.Second
+	cfg.Mempool.TTLDuration = 0 * time.Second
```

Comment:
we removed time based TTLs in the mempool so I'm wondering if we could just ignore this entirely